libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt

```diff
@@ -87,8 +97,9 @@ class MapboxDirectionsSession(
         }
         router.getRoute(routeOptions, object : Router.Callback {
```

Comment:
I think it won't work clean, believe cannot just call `getRoute` with `routeOptions` that was stored on first `getRoute` request. In this case you always get faster route from `origin` and it's not that you need.  Ptal at [RouteFetcher, line 100](https://github.com/mapbox/mapbox-navigation-android/blob/1bf6dee6d0ee6a413e7339b13d86c3f50b4d31a4/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/route/RouteFetcher.kt#L100). Here new RouteOptions is created based on route progress and location

Reply:
Yes this was the original placeholder implementation. This issue is already captured [here](https://github.com/mapbox/navigation-sdks/issues/221) and I have started looking into it as we speak. Also, we don't have an equivalent of `RouteFetcher` in v1.0. Please keep an 👀 on this [ticket](https://github.com/mapbox/navigation-sdks/issues/221) for further info. 😄